controls/roles/validator-import/tasks/main.yml

```diff
@@ -0,0 +1,176 @@
+      content: "{{ item.content }}"
+      owner: "{{ stereum_service_configuration.user }}"
+      group: "2000"
+      mode: 0700
+    become: yes
+    with_items: "{{ validator_keys }}"  
+
+  - name: Creating expect script for import 
+    copy:
+      dest: "/opt/app/import/lighthouse_validator_import.sh"
+      content: |
+        #!/usr/bin/expect
+
+        set timeout -1
+        set validator_password [lindex $argv 0];
+
+        if {[ llength $validator_password ] == 0 } {
+           exit
+        }
+
+        spawn sh -c { sudo docker run --rm -it \
+                       -e DEBUG_LEVEL=debug \
+                       -e NETWORK=prater \
+                       -e DATADIR="/opt/app/validator" \
+                       -e DIRECTORY="/opt/app/launchpad" \
+                       -v "/opt/app/services/2e83d610-48b8-11ec-802a-330dee8e6b2e/validator":"/opt/app/validator" \
+                       -v "/opt/app/services/2e83d610-48b8-11ec-802a-330dee8e6b2e/launchpad":"/opt/app/launchpad" \
+                       --user "2000":"2000" \
+                       --net "stereum" \
+                       --entrypoint "/opt/app/start/import-launchpad-wallet.sh" \
+                       validator_image
+                    }
+
+        expect {
+            "Enter the keystore password" {
+              send "$validator_password\r"
+              sleep 3
+              set results $expect_out(buffer)
+              if {[regexp -nocase "Invalid password" $results]} {
+                exit
+              } elseif {[regexp -nocase "Skipping import of keystore for existing public key" $results]} {
+                exit
+              }
+              exp_continue
+            }
+        } 
+      owner: "{{ stereum_service_configuration.user }}"
+      group: "2000"     
+      mode: 0700
+    become: yes   
+
+  - name: Start beacon service
+    docker_container:
+      command_handling: correct
+      name: beacon
+      hostname: beacon
+      user: "{{ stereum_service_configuration.user }}"
+      image: "{{ stereum_service_configuration.image }}"
+      env: 
+            DEBUG_LEVEL: debug
+            NETWORK: prater
+            ETH1_NODES: https://goerli.infura.io/v3/20fd3ed2418742e9b727857e32b40f9c
+            DATADIR: /opt/app/beacon
+            SLASHERDIR: /opt/app/slasher
+            SLASHER_DB_SIZE: "16"
+      ports:
+        - 9000:9000/tcp
+        - 9000:9000/udp 
+        - 127.0.0.1:5052:5052     
+      command: []
+      entrypoint: ["/opt/app/start/beacon.sh"]
+      restart_policy: "unless-stopped"
+      state: started
+      # recreate: yes # default: no
+      networks:
+        - name: stereum
+      # working_dir: "{{ stereum_service_configuration.working_dir }}" # doesn't work at all
+      volumes: "{{ stereum_service_configuration.volumes }}"
+      log_driver: "json-file"
+      log_options:
+        max-file: "10"
+        max-size: "100m"
+    become: yes               
+
+  - name: Start validator service
```

Comment:
same as beacon, please run in converge.yaml

Reply:
done!